sw/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { setUpServiceWorker } from './service-worker/service-worker.js'
+export { OfflineInterface } from './offline-interface/offline-interface.js'
+export { checkForUpdates } from './lib/registration.js'
```

Comment:
`setUpServiceWorker` will be used in the app shell,
`OfflineInterface` is instantiated in the app adapter,
`checkForUpdates` is used both in the offline interface and potentially in the app adapter, to enable the fatal error boundary to update a service worker upon refresh